go/vt/vttablet/onlineddl/executor.go

```diff
@@ -2230,48 +2286,67 @@ func (e *Executor) executeMigration(ctx context.Context, onlineDDL *schema.Onlin
 	return nil
 }
 
+// runNextMigration picks one 'ready' migration that is able to run, and executes it.
+// Possible scenarios:
+// - no migration is in 'ready' state -- nothing to be done
+// - a migration is 'ready', but conflicts with other running migrations -- try another 'ready' migration
+// It is therefore possible that there is a 'ready' migration, and still this function only handles one.
```

Comment:
This sentence is confusing. I would have thought that it is possible that there is a "ready" migration and still this function chooses none.

Reply:
Oh wow the wording was really bad. Changed to:

```
// Note that per the above breakdown, and due to potential conflicts, it is possible to have one or
// more 'ready' migration, and still none is executed.
```